### ISSUE TITLE
[Enhancement] Docs - banner type recipe on card showcase

### ISSUE THREAD

**Issue by RasmusKjeldgaard** [kirbydesign/designsystem#3248]
- [ ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
A common use-case for the card is to act as a "banner" with a background image and a kirby-card-footer with a disclosure arrow that shows that the card is clickable. 

We should add a section to the card docs with an example like below, where the cards' width and height is not defined by content as in the other examples. Instead we should show how the height and width is defined in css, when no text-content is inside the card, but a certain size is still needed  

```
<kirby-card class="banner" [hasPadding]="true">
  <kirby-card-footer [hasPadding]="false">
    <kirby-item disclosure="arrow-more">
      Banner call to action
    </kirby-item>
  </kirby-card-footer>
</kirby-card>
```

The background-image example can be used as inspiration, with the addition of a `kirby-card-footer` with some example content and a disclosure arrow.

The end product could be something like this:
<img width="707" alt="image" src="https://github.com/kirbydesign/designsystem/assets/42470636/481ad33d-d4d0-4a19-8937-d65ef943981a">

Example image: https://images.unsplash.com/photo-1515876305430-f06edab8282a

## Are there any additional context?
Please note that I _think_ we need to add a tiny bit of css to the card component for the card-footer to position itself correctly. Most likely we need to add `height: 100%;` to the content-layer inside the card, otherwise the footer will not flow to the bottom.

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
